packages/holocron/__tests__/loadModule.web.spec.js

```diff
@@ -143,11 +149,11 @@ describe('loadModule.web', () => {
       })
     );
     const loadError = new Error('failed to load module');
-    mockElement.onerror(loadError);
-    await expect(loadPromise).rejects.toBe(loadError);
+    expect(mockElement.addEventListener.mock.calls[0][1](loadError)).toBeUndefined();
```

Comment:
might want to assert that this is the listener for the call to `addEventListener` with `'error'`?

Reply:
is this what you meant? let me know if this works or if you have any other suggestions

Reply:
Also, should we be refactoring in this effort when the purpose is to update for linting rules? We should create separate refactor efforts if we wish to make fixes.